ocaml/idl/ocaml_backend/dune

```diff
@@ -1,5 +1,5 @@
 (executable
-  (modes exe)
+  (modes byte)
```

Comment:
why is running it earlier in the build process help with speeding the build process?

Reply:
Using `dune build --trace-file` you can see some of this in the Perfetto UI. (not the dependencies, but just look for long bars and click on to see the filename)